examples/ex_fiff_sniffer/main.cpp

```diff
@@ -12,6 +12,17 @@
 #include <fiff/fiff_stream.h>
 #include <fiff/c/fiff_digitizer_data.h>
 
+////////////////////////
+
+#include <connectivity/network/network.h>
```

Comment:
Connectivity dependency?

Reply:
Sorry I am in the process of throwing a bunch of code in this example's main() to test various points of dealing with the digitizer data without reading from file. These changes are not intended to be merged.

Reply:
That being said, this is testing the part of the hpi display code that needed the file, and it works with data found in the isotrak file, so this is good news.

Reply:
:)